pageserver/src/walredo.rs

```diff
@@ -822,7 +831,7 @@ impl PostgresRedoProcess {
                 // good enough, the important thing is to get the message to the log.
                 if n > 0 {
                     error!(
-                        "wal-redo-postgres: {}",
+                        "wal-redo-postgres: {}, ",
```

Comment:
I don't understand the extra comma here

Reply:
Sorry, was just artifact of removed message - will undo it.